.rubocop.yml

```diff
@@ -51,6 +51,7 @@ Style/Documentation:
   Exclude:
     - 'config/application.rb'
     - '**/application_*.rb'
+    - 'app/policies/**/*.rb'
```

Comment:
Are these classes just clear enough to not worry about this rule?

Reply:
There is nothing useful to be said in the documentation about a policy. `WorkPolicy` = "A policy for a Work"